recipe/meta.yaml

```
@@ -22,20 +22,19 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
-    - pyct 0.5.0
-    - param 2.0
+    - hatchling
+    - hatch-vcs
   run:
     - python
+    - bokeh >=3.1
     - colorcet
-    - numpy >=1.0
+    - matplotlib-base >=3.0
+    - numpy >=1.21
     - packaging
+    - pandas >=1.3
     - panel >=1.0
-    - pandas >=0.20.0
-    - param >=1.12.0,<3.0
-    - pyviz_comms >=0.7.4
-    - matplotlib-base >=3
+    - param >=2.0,<3.0
+    - pyviz_comms >=2.1
```

```suggestion
    - pyviz_comms >=2.1
  run_constrained:
    - plotly >=4.0
```
https://github.com/holoviz/holoviews/blob/v1.19.0/pyproject.toml#L49